frontend/src/components/custom-tools/prompt-card/PromptCardItems.jsx

```diff
@@ -490,11 +499,29 @@ function PromptCardItems({
                                       : "collapsed-output"
                                   }
                                 >
-                                  {displayPromptResult(
-                                    result.find(
-                                      (r) => r?.profileManager === profileId
-                                    )?.output,
-                                    true
+                                  {!result.find(
+                                    (r) => r?.profileManager === profileId
+                                  )?.output ? (
+                                    <Typography.Text
+                                      style={{
```

Comment:
Inline css needs to be moved to a css file.

Reply:
Done :+1:

Reply:
Done :+1: